pkg/logging/text_logger.go

```diff
@@ -0,0 +1,387 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
```

Comment:
Please check my comments about the copyright
https://github.com/zhaojizhuang/nerdctl/commit/c8c2a66e74725e01029c9b30fe64ec950f2a1637#r96046932

Reply:
ok